clean-abap/CleanABAP.md

```diff
@@ -459,6 +460,23 @@ for example prefer `detection_object_types` to something cryptic like `dobjt`.
 
 > Read more in _Chapter 2: Meaningful Names: Use Pronounceable Names_ of [Robert C. Martin's _Clean Code_]
 
+### Use snake_case
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Names](#names) > [This section](#use-snake_case)
+
+ABAP is case insensitive and limits the number of available letters for naming all kind of DDIC-objects
+and variables. For exampe the maximum length of a method name is 30 characters. The defacto naming convention
+in ABAP is `snake_case`, because camelCase is overwritten when the Pretty Printer is activated and not all team
+members use the 'Keep Camel Case Identifiers' setting in the ADT ABAP Formatter. Also `camelCase` doesn't
+affect DDIC-objects.
+We recommend using `snake_case` consistently instead of `flatcase` or `UPPERCASE`. When you reach the maximum 
+length of an object, try to conscientiously use abbreviations (also see [Avoid abbreviations](#avoid-abbreviations)).
```

Comment:
```suggestion
length of an object, try to conscientiously use abbreviations (see [Use same abbreviations everywhere](#use-same-abbreviations-everywhere)).
```

Reply:
Or are you not actually using "conscientiously" in the sense of always using the same abbreviations? If you mean something else, then maybe a different word would fit better.